maven-plugin/plugin-core/src/main/java/org/jvnet/jaxb/maven/RawXJC2Mojo.java

```diff
@@ -479,6 +481,7 @@ protected void doExecute() throws MojoExecutionException {
 			setupDirectories();
 			doExecute(options);
 			addIfExistsToEpisodeSchemaBindings();
+            deleteEpisodeIfOnlyFileOnEmptyContext();
```

Comment:
Minor indent fix needed

Reply:
Indent is OK here (spaces only), we should fix all the others but out of scope here. Do you want me to do it with tabs
 ?

Reply:
Hrmm.. I think the .editorconfig must not be applied all the way. We should probably use spaces to have single whitespace character type across all data (.java, xml, xsd, etc).

4 - spaces Java code
2 - spaces xml, xjb, xsd, etc.

Resolving this conversation, as it is an .editorconfig issue not a PR issue.